lib/helper_method.rb

```diff
@@ -0,0 +1,110 @@
+module Hotel
+  class Helper_Method
```

Comment:
In software, one existing design pattern that this class and its responsibility might align with/fit with is the idea of a Utility Class. I might call this `HotelUtility` instead of `Helper_Method`.

Also, naming convention in Ruby for classes is TitleCase/PascalCase/CapitalCamelCase, so a better name than `Helper_Method` is `HelperMethod`!